### ISSUE TITLE
restricted shell bug

### ISSUE THREAD

**Issue by dmio** [netblue30/firejail#887]
It seems, that after 0.9.34 version, firejail unable to execute commands with args thru restricted shell.
Examples:
```
# cat /etc/passwd|grep firejail
test:x:1001:1001::/home/test:/usr/bin/firejail
# su -l test -c "ls -l"
Autoselecting /bin/bash as shell
Command name #ls#
...
Running 'ls -l'  command through /bin/bash
execvp argument 0: /bin/bash
execvp argument 1: -c
execvp argument 2: 'ls -l' 
/bin/bash: ls -l: command not found
# firejail -c "date && date"
/bin/bash: date && date: command not found
```
Same with ssh:
```
% ssh test@localhost ls -l
/bin/bash: ls -l: command not found
```
Git over ssh:
```
% LANG=c git push origin
/bin/bash: git-receive-pack '/home/git/my/xyz.git': No such file or directory
fatal: Could not read from remote repository.
```

**Comment by chtaylo2** [comment]
+1 for this issue.  I'd like to use firejail for a remote kdump server, so being able to execute remote commands would be great. Thanks for the great work on this tool.

**Comment by jsquyres** [comment]
See #4326 for a possible solution.

**Comment by rusty-snake** [comment]
This issue is fixed by #4326 as I understand.
